.circleci/get-review-app-url.sh

```diff
@@ -1,17 +1,19 @@
 #!/usr/bin/env bash
 
-GITHUB_DEPLOYMENTS_URL="https://api.github.com/repos/ministryofjustice/hmpps-book-secure-move-frontend/deployments"
+GITHUB_DEPLOYMENTS_URL="https://api.github.com/repos/ministryofjustice/hmpps-book-secure-move-frontend/deployments?ref=${CIRCLE_SHA1}"
 
 while true
 do
-  DEPLOYED_SHA=$(curl --silent -XGET -H "Authorization: token ${GITHUB_API_TOKEN}" $GITHUB_DEPLOYMENTS_URL?ref=${CIRCLE_SHA1} | jq -r '.[0].sha')
+  DEPLOYMENTS=$(curl --silent -XGET -H "Authorization: token ${GITHUB_API_TOKEN}" $GITHUB_DEPLOYMENTS_URL)
+  DEPLOYED_SHA=$(echo $DEPLOYMENTS | jq -r '.[0].sha')
   if [ "$DEPLOYED_SHA" = "$CIRCLE_SHA1" ]; then
     break
   fi
   sleep 5
 done
 
-GITHUB_DEPLOYMENT_STATUS_URL=$(curl --silent -XGET -H "Authorization: token ${GITHUB_API_TOKEN}" $GITHUB_DEPLOYMENTS_URL | jq -r '.[0].statuses_url')
+GITHUB_DEPLOYMENT_STATUS_URL=$(echo $DEPLOYMENTS | jq -r '.[0].statuses_url')
+echo $GITHUB_DEPLOYMENT_STATUS_URL
```

Comment:
`echo $GITHUB_DEPLOYMENT_STATUS_URL`

Sorry - this isn't needed. It was just my debugging and will still cause it to fail